var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -10,8 +10,10 @@ class Charliecloud(MakefilePackage):
     """Lightweight user-defined software stacks for HPC."""
 
     homepage = "https://hpc.github.io/charliecloud"
-    url      = "https://github.com/hpc/charliecloud/archive/v0.2.4.tar.gz"
+    url = "https://github.com/hpc/charliecloud/archive/v0.2.4.tar.gz"
```

Comment:
I would use the `url` for the newer versions so that `spack versions` picks up more results.